src/sentry/quotas/redis.py

```diff
@@ -34,19 +45,39 @@ class RedisQuota(Quota):
     grace = 60
 
     def __init__(self, **options):
-        self.cluster, options = get_cluster_from_options("SENTRY_QUOTA_OPTIONS", options)
+        self.is_redis_cluster, self.cluster, options = get_dynamic_cluster_from_options(
+            "SENTRY_QUOTA_OPTIONS", options
+        )
+
+        # NB: RedisCluster is a client, whereas RB needs to resolve one
```

Comment:
This comment is pretty difficult to understand unless you're pretty familiar with both implementations — it could probably be clearer.